### DISCUSSION TITLE
lightbox.loadAndOpen(0) opens empty lightbox

### DISCUSSION THREAD

**Discussion by emollini** [dimsemenov/PhotoSwipe#2020] [Q&A] [answered]
Hello there.

I made a photoswipe lightbox which is working fine and opens when I click on the thumbnails. But when I try to open it with the loadAndOpen(0) method (I want to open it on page load), the opened lightbox stays empty, no images inside (the _pswp__container_ contains 3empty _pswp__item_-classed divs).

Photoswipe Version 5.3.6

Any help?

**Comment by dimsemenov** [comment, marked as answer]
If you use `gallery` and `children` options, `loadAndOpen` doesn't know which exactly gallery to use, since there might be multiple galleries on a page. Instead you can do something like:

```js
lightbox.loadAndOpen(0, {
  gallery: document.querySelector('.your-gallery-selector');
});
```

or just trigger click on the element that you wish to open

**Reply by emollini** [reply]
That did it, thank you!
